modules/core/src/smithy4s/Service.scala

```diff
@@ -35,12 +35,71 @@ import kinds._
   *   metaprogramming.
   */
 trait Service[Alg[_[_, _, _, _, _]]] extends FunctorK5[Alg] with HasId {
+  /**
+   * A datatype (typically a sealed trait) that reifies an operation call within
+   * a service. It essentially captures the input and type indexes that the operation
```

Comment:
Shall we also mention that the operation also captures the input _value_?

Reply:
also I really like telling people that Operation is an "applied endpoint". Perhaps we can use that here as well.

Reply:
[0b5591a](https://github.com/disneystreaming/smithy4s/pull/908/commits/0b5591adaa6ffbaff1c0df45adad24178ac2e198)